js/plugins/iq.js

```diff
@@ -160,11 +160,21 @@ var IqPlugin= function (framework)
       case OO.Analytics.EVENTS.VIDEO_PLAYER_CREATED:
         if (params && params[0])
         {
-          pcode = params[0].pcode;
-          playerId = params[0].playerBrandingId;
+          eventParams = params[0];
+          pcode = eventParams.params.pcode;
```

Comment:
Can we add a null check to see if eventParams.params exists?

Reply:
I added it as a check on params[0].params, since params[0] == eventParams